src/channel.ts

```diff
@@ -414,9 +414,12 @@ export class Channel<
     if (!reaction || Object.keys(reaction).length === 0) {
       throw Error(`Reaction object is missing`);
     }
+
+    const { skip_push, ...rest } = reaction;
```

Comment:
a bit weird to put this into the reaction object, maybe we should change the function signature to `sendReaction(id, reaction, options)`? we have a major release coming up so we can unify this across all similar function making them future proof

Reply:
nice, if we can change signature, I would prefer it too.

Reply:
what do you think @vishalnarkhede?

Reply:
will do this change and will also introduce options to `sendMessage` for the same reason (not to mix control keys with custom fields). Sounds good ?

Reply:
sounds good

Reply:
updated